data/json/items/resources/tailoring.json

```
@@ -85,7 +85,7 @@
     "id": "scrap_lycra",
     "copy-from": "scrap_cotton",
     "type": "GENERIC",
-    "name": { "str_sp": "Lycra scraps" },
+    "name": { "str_sp": "lycra scraps" },
```

```suggestion
    "name": { "str_sp": "Lycra scraps" },
```

bring that back, it's the same as with Kevlar